### ISSUE TITLE
Region protections no longer work.

### ISSUE THREAD

**Issue by andrewkm** [Tsuser1/Modern-LWC#93]
**/cprivate region:spawn**
Users part of spawn region cannot open this chest.
Only the person that ran the /cprivate command can.

Setup is all latest as of this post.
ModernLWC: 2.1.3 Build 10
WE: 7.0.0 Build 4106
WG: 7.0.0 Build 1820
Paper: 1.13.2 Build 566
No console errors.

**Comment by Tsuser1** [comment]
Please provide the command output for the ModernLWC version, it isn’t based on build numbers, but rather, commits.

**Comment by andrewkm** [comment]
Built on this:
https://github.com/Tsuser1/Modern-LWC/commit/aba668e7b5a282e1752e4d1f1b459817e88e76e8

**Comment by pop4959** [comment]
I am not very accustomed to WorldGuard regions, but I tried `/cmodify r:__global__` on a private chest, and made sure that the person trying to access it was either a member or owner in the region, and it did not work. It looks like this might be a bug.

**Comment by pop4959** [comment]
I just tested this again, because I was planning to fix it, but I think my testing last time was incorrect. I did manage to get a region protection to work properly.

Steps:
Select WorldEdit region
/region define regionname
Place a chest (auto-register) in regionname
/cmodify r:regionname
/region addowner regionname player
player can now access private chest with region protection in regionname

Hope that helps. Closing the issue now since it doesn't appear to be an issue.
